pgmpy/factors/CPD.py

```diff
@@ -125,6 +125,79 @@ def __repr__(self):
 
         return var_str + evidence_str + ') at {address}>'.format(address=hex(id(self)))
 
+    def set_parents_order(self, new_order):
+        """
+        Change the order of the parents
+
+        Parameters
+        ----------
+        new_order: list
+            reordered list of evidences.
+        The returned tabular cpd has values reordered according to the new order provided by the user.
+
+        Example
+        --------
+        >>> from pgmpy.factors import TabularCPD
+        >>> final_cpd = TabularCPD('grade', 5, [
+        ... [0.12, 0.6, 0.2, 0.9, 0.5, 0.4, 0.5, 0.5, 0.4, 0.9, 0.8, 0.8],
+        ... [0.12, 0.6, 0.2, 0.9, 0.5, 0.4, 0.5, 0.5, 0.4, 0.9, 0.8, 0.8],
+        ... [0.3, 0.8, 0.4, 0.8, 0.8, 0.8, 0.7, 0.7, 0.2, 0.8, 0.8, 0.4],
+        ... [0.4, 0.9, 0.5, 0.7, 0.3, 0.2, 0.8, 0.8, 0.1, 0.6, 0.8, 0.8],
+        ... [0.5, 0.1, 0.6, 0.6, 0.2, 0.1, 0.9, 0.9, 0.8, 0.5, 0.8, 0.8]],
+        ...                                   evidence=['intel', 'diff', 'iq'], evidence_card=[2, 3, 2])
+        >>> print(final_cpd)
+╒═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╕
```

Comment:
@kislayabhi This make the doc string looks ugly. So instead of 

```python
>>> print(final_cpd)
```

use 
```python
>>> final_cpd
```
It would call the ```__repr__``` method which also shows the ordering of the variables.